sda/processor_test.go

```diff
@@ -114,7 +61,9 @@ func TestProcessor_RegisterMessages(t *testing.T) {
 }
 
 func TestProcessor_GetReply(t *testing.T) {
-	h1 := newHostMock(network.Suite, "127.0.0.1")
+	defer log.AfterTest(t)
```

Comment:
So, do we go with one "global" cleanup per package, or cleanup each test separately?